examples/node/MarketDataSubscription_LongPoll.js

```diff
@@ -75,4 +82,6 @@ function unsubscribe()
     });    
 }
 
+var opts = cmdline.parse({host: 'http-api-host', port: 3000});
```

Comment:
Exactly, defaulting `host` & `port` will just confuse people cause they may think there is one demo server running somewhere, but there isn't. We are on the same page :smile: